packages/lib/action/service.ts

```diff
@@ -122,10 +123,10 @@ export const createAction = async (data: TActionInput): Promise<TAction> => {
     },
   });
 
-  // revalidate cache
   revalidateTag(sessionId);
-  revalidateTag(getActionClassCacheTag(name, environmentId));
-  revalidateTag(getActionsCacheTag(environmentId));
+  actionCache.revalidate({
```

Comment:
Now we only revalidate the actions. But we still need to revalidate the actionClasses cache if a new action class is created in this function. If we can't get the info from the prisma call if the action was newly created, we need to make the check ourself at the start of the function if the actionClass already exists (using the cached actionClass service). And based on this revalidate the actionClass cache or not.

Reply:
Done 🚀